SU-SWS/stanford_profile
# CAW21-82 Build accordion paragraph component

# READY FOR REVIEW

# Summary
- Accordion paragraph component
- https://github.com/SU-SWS/jumpstart_ui/pull/111
- @cjwest please have a quick accessibility review of this please.

# Need Review By (Date)
- 8/10

# Urgency
- medium/high

# Steps to Test
1. checkout this branch and the branch in https://github.com/SU-SWS/jumpstart_ui/pull/111
2. `drush cim -y; drush cr`
3. create a basic page and add an accordion to it
4. save and verify it functions as expected.

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
